substrate/client/basic-authorship/src/basic_authorship.rs

```diff
@@ -290,8 +290,8 @@ where
 			"basic-authorship-proposer",
 			None,
 			Box::pin(async move {
-				// leave some time for evaluation and block finalization (33%)
-				let deadline = (self.now)() + max_duration - max_duration / 3;
+				// leave some time for evaluation and block finalization (20%)
+				let deadline = (self.now)() + max_duration - max_duration / 5;
```

Comment:
from reading https://github.com/paritytech/polkadot-sdk/issues/7974#issuecomment-2739761223, it seems the suggestion is to remove this extra deadline altogether - this can happen separately. for it seems the suggestion is to lower this even further to 10% by default reading the resolved thread above
```suggestion
				// leave some time for evaluation and block finalization (10%)
				let deadline = (self.now)() + max_duration - max_duration / 10;
```

Reply:
Are we okay with sneaking this global change into `master` in the scope of this PR? @bkchr?

Reply:
@skunert @sandreim @eskimor any opinions here? this is the only blocking change